python/lsst/meas/base/dataIds.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
```

Comment:
It's meant to be used by `meas_mosaic`: see https://github.com/HyperSuprime-Cam/meas_mosaic/search?utf8=✓&q=PerTractCcdDataIdContainer
I tried to keep the import as similar as possible (modulo that the file is now in mea_base, not pipe_tasks).  I think it would be odd to have to call it from `forcedPhotCcd.py`, but if you think there's a more appropriate place for it (`pipe_base` perhaps?), I'm happy to move it.